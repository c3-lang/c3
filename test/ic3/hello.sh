#!/bin/sh
$IC3 < hello.in > hello.out 2> hello.err
echo $? > hello.ret
diff -u hello.out.expected hello.out &&
    diff -u hello.err.expected hello.err &&
    diff -u hello.ret.expected hello.ret
