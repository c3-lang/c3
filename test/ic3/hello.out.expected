ic3> "Hello, world !"
"Hello, world !"
ic3> 
